guides/common/modules/proc_configuring-dns-on-smart-proxies.adoc

```diff
@@ -0,0 +1,257 @@
+[id="Configuring_DNS_on_Smart_Proxies_{context}"]
+= Configuring DNS on {SmartProxies}
+
+Activate the DNS management module within your {SmartProxy}.
+This is used to update and remove DNS records from existing DNS zones.
+
+The DNS module can manipulate any DNS server that complies with the ISC Dynamic DNS Update standard and can therefore be used to manage both Microsoft Active Directory and BIND servers. Updates can also be made using GSS-TSIG, see the second section below. Additional providers are available for managing libvirt's embedded DNS server (dnsmasq) and Microsoft Active Directory using dnscmd, for static DNS records, avoiding scavenging.
+
+Builtin providers are:
+
+* `dns_nsupdate` - dynamic DNS update using nsupdate
+* `dns_nsupdate_gss` - dynamic DNS update with GSS-TSIG
+* `dns_libvirt` - dnsmasq DNS via libvirt API
+* `dns_dnscmd` - static DNS records in Microsoft Active Directory
+
+Extra providers are available as plugins and can be installed through packages. See the following pages for more information:
+
+* http://projects.theforeman.org/projects/foreman/wiki/List_of_Smart-Proxy_Plugins[List of smart proxy plugins]
+* link:%7B%7B%20site.baseurl%20%7D%7Dplugins/[Plugin installation documentation]
```

Comment:
This looks like something went wrong in the conversion.

Reply:
Fixed.